src/main.cpp

```diff
@@ -7114,13 +7113,6 @@ bool static ProcessMessage(const CChainParams& chainparams, CNode* pfrom, string
             }
         }
 
-        // Relay alerts
-        {
-            LOCK(cs_mapAlerts);
-            for (std::pair<const uint256, CAlert>& item : mapAlerts)
-                item.second.RelayTo(pfrom);
-        }
```

Comment:
It is safe to remove this because alerts already would not have been relayed with `fAlerts` false.